client/rpc/block_converter.go

```diff
@@ -0,0 +1,115 @@
+package rpc
+
+import (
+	"encoding/json"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+type ResultBlock struct {
+	BlockMeta BlockMeta `json:"block_meta"`
+	Block     Block     `json:"block"`
+}
+
+type BlockMeta struct {
```

Comment:
nitpick, to follow the same structure as header. `types.BlockMeta`, not blocking the merge though.

Reply:
Unified all the types. Should be good now